Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: manifests assets unit-tests helm-unit-tests
+.PHONY: manifests assets unit-tests helm-unit-tests bats-unit-tests preflight
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)